gateway/gateway/handler/auth.go

```diff
@@ -497,7 +524,7 @@ func (cx *Context) sessionsHandlerV1Post(w http.ResponseWriter, r *http.Request)
 			Code:        0,
 		}
 		cx.handleErrorJson(w, r, errVC,
-			"provided credentials are not valid", retErr, http.StatusUnauthorized)
+			"provided credentials are not valid", retErr, http.StatusBadRequest)
```

Comment:
This is better for security. Is that why you're changing these to not display an auth failure?

Reply:
I changed this to status code 400, bad request, instead of unauthorized because in this case the validation above is to test if the provided credentials could even be valid credentials. Therefore, the user made a mistake if the credentials provided aren't even valid in the system since it would be impossible for those same credentials to have been added to the system, so the user must have made a mistake.